doc/01-About.md

```diff
@@ -1,8 +1,23 @@
 # Icinga DB
 
-![Icinga DB Context](images/about/icinga-db-in-icinga-context.png)
+Icinga DB is a set of components for publishing, synchronizing and
+visualizing monitoring data in the Icinga ecosystem, consisting of:
 
-Icinga DB serves as a synchronisation daemon between Icinga 2 (Redis) and Icinga Web 2 (MySQL/MariaDB/PostgreSQL database).
-It synchronises configuration, volatile states and history of an Icinga 2 environment using checksums.
+* The Icinga DB daemon, which synchronizes monitoring data between a Redis server and a database
+* Icinga 2 with its [Icinga DB feature](https://icinga.com/docs/icinga-2/latest/doc/14-features/#icinga-db) enabled,
+  responsible for publishing the data to the Redis server, i.e. configuration and its runtime updates, check results,
+  state changes, downtimes, acknowledgements, notifications, and other events such as flapping 
+* And Icinga Web with the
+  [Icinga DB Web](https://icinga.com/docs/icinga-db-web) module enabled,
+  which connects to both Redis and the database to display and work with the most up-to-date data
 
-Icinga DB also supports reading from multiple environments and writing into a single database.
+![Icinga DB Architecture](images/icingadb-architecture.png)
+
+## Installation
+
+To install Icinga DB see [Installation](02-Installation.md).
+
+## License
+
+Icinga DB and the Icinga DB documentation are licensed under the terms of the
+[GNU General Public License Version 2](../LICENSE).
```

Comment:
Works on GitHub, 404 in the rendered version.

Reply:
Okay, so according to @bobapple, this link doesn't work on icinga.com/docs due to some special casing for the `01-About.md` file as it's used as the index, so probably best to use an absolute link (for now). This also explains why the link to `config.example.yml` works in the other file.